typescript/sdk/src/middleware/accounts.hardhat-test.ts

```diff
@@ -67,7 +67,9 @@ describe('InterchainAccountRouter', async () => {
       localDomain,
       signer.address,
     );
-    await local.dispatch(remoteDomain, [{ to: recipient.address, data }]);
+    await local['dispatch(uint32,(address,bytes)[])'](remoteDomain, [
```

Comment:
I have tended to avoid these overrides bc they are awkward from SDK but I don't see either of these libs being used from there so 🤷‍♂️